README.md

```diff
@@ -20,8 +20,9 @@ docker run --rm aws-cost-slack-reporter
 accounts YAML file with AWS credentials:
 
 ```yaml
-account-name:
-  secret_access_key: "Secret Access Key" # pragma: allowlist secret
-  access_key_id: "Access Key ID" # pragma: allowlist secret
+accounts:
```

Comment:
`accounts` is redundant, `account name` is the key

Reply:
wrong; we are iterating over `accounts` key items, see https://github.com/RedHatQE/aws-cost-reporter/blob/f4082b7d56c55e6f3cc99f674eba37565c09309d/app/reporter.py#L43